SU-SWS/decanter
# 89 browsersync

# READY FOR REVIEW

# Summary
- add `browserSync`

# Needed By (Date)
- Whenever developers get tired of manually reloading their browsers

# Urgency
- Lower than low

# Steps to Test

1. check out the `89-browsersync` branch
1. `npm install` - add the grunt-browser-sync package
1. `grunt dev` - run the new `dev` task, which
    - (re)builds the styleguide initially
    - launches browserSync, which opens a browser window to port 3000 (we can change the port)
    - watches your source files, including `.scss`, `.twig` and `.json`

# Affected Projects or Products
- Nobody but us devs

# Associated Issues and/or People
- #89 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
